secure-tomcat-image/src/main/docker/server.xml

```diff
@@ -138,9 +138,9 @@
                 <!-- Access log processes all example.
                      Documentation at: /docs/config/valve.html
                      Note: The pattern used is equivalent to using pattern="common" -->
-                <Valve className="org.apache.catalina.valves.AccessLogValve" directory="logs"
-                       prefix="localhost_access_log." suffix=".txt"
-                       pattern="%h %l %u %t &quot;%r&quot; %s %b" />
+                <!--<Valve className="org.apache.catalina.valves.AccessLogValve" directory="logs"-->
+                       <!--prefix="localhost_access_log." suffix=".txt"-->
+                       <!--pattern="%h %l %u %t &quot;%r&quot; %s %b" />-->
```

Comment:
I turned off generic Tomcat access logging.  When running through an issue with Chris McCann, we noticed the ART Tomcat access log was ~24GB.